runners/apex/src/test/java/org/apache/beam/runners/apex/ApexYarnLauncherTest.java

```diff
@@ -134,6 +134,7 @@ public void testCreateJar() throws Exception {
       Assert.assertTrue("manifest", Files.isRegularFile(zipfs.getPath(JarFile.MANIFEST_NAME)));
       Assert.assertTrue("file1", Files.isRegularFile(zipfs.getPath(file1)));
     }
-
+    // delete the directory to not let stuff around after the test execution
+    FileUtils.deleteDirectory(baseDir);
```

Comment:
<!--new_thread; commit:892fd27f1c4b72ffc3a9ac9c19bcac1c66492bea; resolved:0-->
Probably this should use `@Rule TemporaryFolder`